docs/source/examples/smoothing.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>I think we should recommend explicitly using <code>{'lon': 5, 'lat': 5}</code> since it's not intuitive that this automatically does 5 deg for both. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>